packages/playground/tests/e2e/cards/kcpUtils.js

```diff
@@ -0,0 +1,49 @@
+import { setIframeSelector, fillIFrame, deleteFromIFrame, checkIframeContainsValue } from '../commonUtils';
+
+import { KOREAN_TEST_CARD, NON_KOREAN_TEST_CARD, TEST_DATE_VALUE, TEST_CVC_VALUE, TEST_PWD_VALUE, TEST_TAX_NUMBER_VALUE } from '../constants';
+
+/**
+ * Unique to each component are where the iframes are to be found,
+ * the indices by which a specific iframe can be identified,
+ * and the selectors for elements found within it
+ */
+setIframeSelector('.card-field iframe');
+
+/**
+ * @param t - TestController ref
+ * @param value - text to enter into selected element in iframe
+ * @param replace - boolean: whether typed text will replace existing content
+ * @returns {Promise<*>}
+ */
+export const fillCardNumber = async (t, value = NON_KOREAN_TEST_CARD, replace = false) => {
+    return fillIFrame(t, 0, '#encryptedCardNumber', value, replace);
+};
+
+export const deleteCardNumber = async t => {
+    return deleteFromIFrame(t, 0, '#encryptedCardNumber');
+};
+
+export const fillDate = async (t, value = TEST_DATE_VALUE, replace = false) => {
+    return fillIFrame(t, 1, '#encryptedExpiryDate', value, replace);
+};
+
+export const fillCVC = async (t, value = TEST_CVC_VALUE, replace = false) => {
+    return fillIFrame(t, 2, '#encryptedSecurityCode', value, replace);
+};
+
+export const fillDateAndCVC = async (t, dateValue = TEST_DATE_VALUE, cvcValue = TEST_CVC_VALUE) => {
+    await fillDate(t, dateValue);
+    return fillCVC(t, cvcValue);
+};
```

Comment:
Nice helper functions! Should we maybe move them into some more generic `cardUtils.ts` file, since they are not exclusive to Korean cards?

Reply:
But they kind of are exclusive as I say in the comment. 
Only KCP knows of an iframe index 3 that is a password field.
Likewise a giftcard would have a different index for its securityCode field than a regular card.
And a stored card has a different kind of field at index 1 than a regular card.
Maybe some of it can be abstracted but I see this as a work in progress and this kind of thing can be done later, as we all work on it